tests/types/bazel-test.json

```diff
@@ -83,7 +85,7 @@
         },
         "subpath": null
       },
-      "expected_output": "pkg:bazel/rules_java@7.8.0?repository_url=https://example.org/bazel-registry",
+      "expected_output": "pkg:bazel/rules_java@7.8.0?repository_url=https:%2F%2Fexample.org%2Fbazel-registry",
```

Comment:
If the input is transformed to correct it then this needs to be "test_group": "advanced"